app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PostsController < ApplicationController
+  before_action :authenticate_user!
+
```

Comment:
add index action and redirect to the index after the post was created

Reply:
Can i redirect to user profile page (action show in users controller), where all user's posts will be shown?

Reply:
I think better will be index action here (all posts include user's posts)